### PR TITLE
Update iOS assets collection to return filename, rotation, mime type, and file size.

### PR DESCRIPTION
**Pull request by billscheidel**
There is a common set of information needed for most files that is currently missing from the images returned by the camera roll.  Currently the details must be retrieved separately by stating the files but not all information is available that way.  When I took a look at the iOS implementation, all of the data is already available so I just updated the asset collection to return it.  

I've only tested this code on iOS 12.2.

### CODE REVIEW

File: ios/RNCCameraRollManager.m [hunk 1]
```diff
@@ -287,8 +287,12 @@ static void RCTResolvePromise(RCTPromiseResolveBlock resolve,
              @"group_name": [assetCollection localizedTitle],
              @"image": @{
                  @"uri": uri,
+                 @"fileName": [asset valueForKey:@"filename"],
+                 @"originalRotation": [asset valueForKey:@"exifOrientation"],
```
Comment (bartolkaruza) [inline ios/RNCCameraRollManager.m]: originalRotation, type and fileSize are missing from the flow type https://github.com/react-native-community/react-native-cameraroll/pull/30 and the typescript interfaces: https://github.com/react-native-community/react-native-cameraroll/pull/30

**Comment by bartolkaruza** [comment]
@billscheidel Do you have the time to take this PR further and do you need help anywhere?

**Comment by billscheidel** [comment]
Probably not.  I added what I needed and thought I would post it since it seemed like other people could benefit.


On Apr 14, 2019, at 3:02 AM, Bartol Karuza <notifications@github.com<mailto:notifications@github.com>> wrote:


@billscheidel<https://github.com/billscheidel> Do you have the time to take this PR further and do you need help anywhere?

**Comment by vforvasile** [comment]
Great job guys, can this be pushed forward?

**Comment by dokkis** [comment]
@bartolkaruza I took over this PR in here to include at least the file size: https://github.com/react-native-community/react-native-cameraroll/pull/180

I didn't included orientation as in Android `https://developer.android.com/reference/android/provider/MediaStore.MediaColumns#ORIENTATION` is added from API level 29.

**Comment by bartolkaruza** [comment]
Closing this in favor of #180

**Comment by Rotemy** [comment]
So there is no way to know if an image or video is rotated in Android?

**Pull request closed without merging**